conda_pack/formats.py

```diff
@@ -28,7 +28,7 @@ def _parse_n_threads(n_threads=1):
     return n_threads
 
 
-def archive(fileobj, path, arcroot, format, compress_level=4, zip_symlinks=False,
+def archive(output, fileobj, path, arcroot, format, compress_level=4, zip_symlinks=False,
             zip_64=True, n_threads=1, verbose=False):
```

Comment:
Can you please make this change non-breaking?
```suggestion
def archive(fileobj, path, arcroot, format, compress_level=4, zip_symlinks=False,
            zip_64=True, n_threads=1, verbose=False, output=None):
```